changelog/4709.doc.rst

```diff
@@ -0,0 +1 @@
+Created the new gallery example :ref:`sphx_glr_generated_gallery_units_and_coordinates_north_offset_frame_py` for `~sunpy.coordinates.NorthOffsetFrame`.
```

Comment:
```suggestion
Created the new gallery example :ref:`sphx_glr_generated_gallery_units_and_coordinates_north_offset_frame.py` for `~sunpy.coordinates.NorthOffsetFrame`.
```
Oops again.  Sorry I keep mucking up this suggestion.  This time I have actually tested it first.